fight/doctor.js

```diff
@@ -0,0 +1,27 @@
+const Unit = require('./unit');
+
+function Doctor(){
+    Unit.apply(this, arguments);
+    this._healPower = 10;
+}
+
+Doctor.prototype = new Unit();
+
+Doctor.prototype.heal = function (unit) {
+
+    let bonusAttack = (this._level > 1) ? (this._level * 0.1) : 0;
+
+   // let healthBefore = unit.getHealth();
+    unit.addHealth(this._healPower + bonusAttack);
+   // let healthAfter = unit.getHealth();

```

Comment:
A lot of dev comments, you better clean them up

Reply:
lack of version control system usage - must save all the code :)